src/tables/stat.rs

```diff
@@ -332,4 +332,70 @@ impl<'a> Table<'a> {
             version: self.version,
         }
     }
+
+    /// Returns the first matching subtable for a given axis.  If no match value is given the first
+    /// subtable for the axis is returned.  If a match value is given, the first subtable for the
+    /// axis where the value matches is returned.  A value matches if it is equal to the subtable's
+    /// value or contained within the range defined by the subtable.  If no matches are found `None`
+    /// is returned.  Typically a match value is not specified for non-variable fonts as multiple
+    /// subtables for a given axis ought not exist.  For variable fonts a non-`None` match value
+    /// should be specified as multiple records for the variation axes exist.
+    pub fn subtable_for_axis(
```

Comment:
Ok, so you have decided to create a query method instead. Fine by me.

Can you fix up the docs a bit? Do not use double spaces and make sure the first line is new-line separated. Like:

```rust
    /// Returns the first matching subtable for a given axis.
    /// 
    /// If no match value is given the first subtable for the axis is returned.
    /// 
    /// If a match value is given, the first subtable for the axis
    /// where the value matches is returned.
    /// A value matches if it is equal to the subtable's value
    /// or contained within the range defined by the subtable.
    /// 
    /// If no matches are found `None` is returned. Typically a match value is not specified
    /// for non-variable fonts as multiple subtables for a given axis ought not exist.
    /// For variable fonts a non-`None` match value should be specified
    /// as multiple records for the variation axes exist.
```